conda/trust/__init__.py

```diff
@@ -0,0 +1,240 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2012 Anaconda, Inc
+# SPDX-License-Identifier: BSD-3-Clause
+
+from functools import lru_cache
+from glob import glob
+import json
+from logging import getLogger
+from os import makedirs
+from os.path import basename, isdir, join, exists
+import warnings
+
+from ..base.context import context
+from ..common.url import join_url
+from ..gateways.connection import HTTPError, InsecureRequestWarning
+from ..gateways.connection.session import CondaSession
+from .constants import INITIAL_TRUST_ROOT, KEY_MGR_FILE
+
+try:
+    from conda_content_trust.authentication import verify_root, verify_delegation
+    from conda_content_trust.common import (
+        load_metadata_from_file,
+        write_metadata_to_file,
+        SignatureError,
+    )
+    from conda_content_trust.signing import wrap_as_signable
+except ImportError:
+    # SignatureStatus.enabled handles this
+    pass
+
+
+log = getLogger(__name__)
+
+
+class _SignatureVerification:
```

Comment:
Since I usually worry about import time side-effects, could we move this into conda.trust.verification or something so when we add additional submodules to conda.trust we're not triggering the imports for conda-content-trust?